src/parameters/all_parameters.cpp

```diff
@@ -274,15 +285,20 @@ void AllParameters::declare_parameters (dealii::ParameterHandler &prm)
                       "  euler | "
                       "  mhd |"
                       "  navier_stokes |"
+                      "  navier_stokes_channel_flow_constant_source_term | "
```

Comment:
should add the option if doing constant or unsteady source term for the channel flow with default as the constant one